### PR TITLE
Use faker to generate psy

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

**Review by sandrica89 — APPROVED** [review]
We could harmonize by using faker random features (See https://fakerjsdocs.netlify.app/api/random.html) but it's already much better than before :)

**Comment by desoindx** [comment]
> We could harmonize by using faker random features (See https://fakerjsdocs.netlify.app/api/random.html) but it's already much better than before :)

I extended the usage to almost every other random place !

File: package.json [hunk 1]
```diff
@@ -62,6 +63,7 @@
     "express-session": "^1.17.2",
     "express-slow-down": "^1.4.0",
     "express-validator": "^6.12.0",
+    "faker": "^5.5.3",
```
Comment (sandrica89) [inline package.json:66]: Shouldn't these be in dev dependencies?

Comment (desoindx) [reply package.json:66]: not really, the dev dependencies are used for pre release stuff. The way our code is done allow for the npm run seed to be run anytime

**Pull request merged**